Bootstrap.mak

```diff
@@ -89,7 +89,7 @@ bsd: $(SRC)
 	$(CC) -o build/bootstrap/premake_bootstrap -DPREMAKE_NO_BUILTIN_SCRIPTS -DLUA_USE_POSIX -DLUA_USE_DLOPEN -I"$(LUA_DIR)" -I"$(LUASHIM_DIR)" $? -lm
 	./build/bootstrap/premake_bootstrap embed
 	./build/bootstrap/premake_bootstrap --to=build/bootstrap gmake
-	$(MAKE) -C build/bootstrap -j`getconf _NPROCESSORS_ONLN` config=$(CONFIG)
+	$(MAKE) -C build/bootstrap -j`getconf _NPROCESSORS_ONLN || getconf NPROCESSORS_ONLN` config=$(CONFIG)
```

Comment:
Would it be possible to remove `_NPROCESSORS_ONLN` or does this depend on the BSD flavour?

Reply:
Actually, it seems `_NPROCESSORS_ONLN` doesn't work on FreeBSD either; so the underscore-less variant would be fine for both OpenBSD and FreeBSD. But as someone else added the variant with underscore in the first place, dunno if it actually works on some *BSD.

Reply:
That would have been me, I just copied the Linux one and it built the project, I didn't investigate the value. If it doesn't work on your FreeBSD it likely didn't work on mine, I think we should remove the underscore version. :+1:

Reply:
Yeah, when it doesn't work it is not easily noticeable with GNU Make, as it accepts `-j` without a numeric value too, and the build doesn't fail. I was (unintentionally) using a non-gnu make on OpenBSD, which then failed because it requires a value for `-j`. And yes I agree, the underscore version should be removed.

Reply:
Cool, can you remove it in this PR? We can merge this once you've done that. 😄

Reply:
Done. 😄